polus-label-to-vector-plugin/README.md

```diff
@@ -0,0 +1,34 @@
+# Label-to-Vector  plugin 
+* This plugin converts a labelled image to vector field.The plugin stores the vector field  as well as the label since the 
```

Comment:
You need to go through your plugin and make sure that formatting is consistent. Remove all double spaces. Don't sure bullet points for everything. The description should be in paragraph format.